elements/pfe-accordion/src/pfe-accordion.js

```diff
@@ -26,9 +26,39 @@ class PfeAccordion extends PFElement {
     return "pfe-accordion.json";
   }
 
-  static get cascadingAttributes() {
+  static get properties() {
     return {
-      on: "pfe-accordion-header, pfe-accordion-panel"
+      color: {
+        title: "Color",
+        type: String
+      },
+      pfeColor: {
+        // @TODO: Deprecate property
+        type: String,
+        prefix: false,
+        alias: "color"
+      },
+      disclosure: {
```

Comment:
I don't see `aria-expanded` in the property definition list

Reply:
I mention this because it was listed as an observed attribute which has been deleted.  If the component still works without it, I'm cool leaving it off.